File: FragmentsView/app/src/main/java/com/bridgelabz/util/fragmentsview/MainActivity.java
package com.bridgelabz.util.fragmentsview;

import android.app.FragmentManager;
import android.app.FragmentTransaction;
import android.content.res.Configuration;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import static android.R.attr.fragment;
import static android.R.attr.id;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        FragmentManager fragmentManager = getFragmentManager();

        FragmentTransaction fragmentTransaction = fragmentManager.beginTransaction();

        Configuration configuration = getResources().getConfiguration();

        if(configuration.orientation == Configuration.ORIENTATION_LANDSCAPE){
            FragmentsLandscap fragmentsLandscap = new FragmentsLandscap();

            fragmentTransaction.replace(android.R.id.content, fragmentsLandscap);
        }
        else{
            FragmentsPotrint fragmentsPotrint = new FragmentsPotrint();

            fragmentTransaction.replace(android.R.id.content, fragmentsPotrint);
        }

        fragmentTransaction.commit();

        //setContentView(R.layout.activity_main);
    }
}

File: kotlin/FragmentsGoogle/app/src/main/java/com/bridgelabz/fragmentsgoogle/MainActivity.kt
package com.bridgelabz.fragmentsgoogle

import android.app.Fragment
import android.app.FragmentTransaction
import android.os.Bundle
import android.support.v7.app.AppCompatActivity

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        var fragment: Fragment = nextFragment()
        fragmentManager.beginTransaction().replace(R.id.mainFragment, fragment).addToBackStack(null).commit()
    }
}
